expense_manager/pythonScripts/import_xlsx_test.py

```diff
@@ -0,0 +1,176 @@
+        for row in self.source_data:
+            row_values = [str(row.get(col, "No data available")) for col in columns]
+            print(", ".join(row_values))
+
+
+
+class DataPackPrivat(DataPack):
+    def __init__(self, file_name):
+        super().__init__()
+        self.file_name = file_name
+        self.bank_name = "privat"
+        self.headers = []
+        self.headers_map = {}
+        self.source_data = []
+
+
+
+    def load_headers_map(self):
+        short_headers = ['date', 'time', 'category', 'card', 'description', 'card_amount', 'card_currency', 'transaction_amount',
+                         'transaction_currency', 'balance_end', 'balance_currency', 'none']
+        for i in range(len(self.headers)):
+            self.headers_map[self.headers[i]] = short_headers[i]
+
+    def load_data(self):
+        try:
+            workbook = openpyxl.load_workbook(self.file_name)
+            sheet = workbook.active
+
+            # We read the headings from the first line
+            self.headers = [cell.value for cell in sheet[2]]
+            self.load_headers_map()
+
+            # We read all the lines after the headings
+            for row in sheet.iter_rows(min_row=2, values_only=True):
+                # We create a dictionary for each line of data
+                row_data = {}
+                for i, header in enumerate(self.headers):
+
+                    # row_data[header] = row[self.headers.index(header)]
+                    row_data[self.headers_map[header]] = row[self.headers.index(header)]
+
+                self.source_data.append(row_data)
+        except FileNotFoundError as e:
+            print(f"File '{self.file_name}' not found. {e}")
+        except Exception as e:
+            print(f"An error occurred while reading the file: {e}")
+
+    def display_data(self):
+        print(f"Bank statement from {self.bank_name}:")
+        for row in self.source_data:
+            print(row)
+
+    def display_columns(self, columns):
+        # We check whether all the transferred columns are in the headers
+        invalid_columns = [col for col in columns if col not in self.headers]
+        if invalid_columns:
+            print(f"The following columns were not found: {', '.join(invalid_columns)}")
+            return
+
+        # We display column headings
+        print(f"Column data: {', '.join(columns)}")
+
+        # We output data for each line
+        for row in self.source_data:
+            row_values = [str(row.get(col, "No data available")) for col in columns]
+            print(", ".join(row_values))
+
+
+def main():
+    
+    parser = argparse.ArgumentParser(description="Gets file names and paths to them.")
+    
+    parser.add_argument('file1', type=str, help="The path to the first file")
+    parser.add_argument('file2', type=str, help="The path to the first file")
+    
+    args = parser.parse_args()
+
+    file1_path = args.file1
+    file2_path = args.file2
+
```

Comment:
if you read [documentation](https://docs.python.org/uk/3/library/argparse.html#action) , we have key `action`, this help us move out from the CLI signature `some.py <file1> <file2>`  which is extremely uncomfortable _( 1) because it always wayts on two file,  2) cause file1 is always Mono and file2 is always Privat and you have no option to forget about ordering. )_

Look on my code: 

![image](https://github.com/user-attachments/assets/fb8ea2ab-fe07-4638-bf4e-ecae541ab53f)

Look at the usages which is nice comparing to previos signature. `some.py -f <file1> -f <file2> ....` and you got option to filling all this files as many as you want. Look better right?

![image](https://github.com/user-attachments/assets/e4c62250-a418-4a6b-8aad-9c9b7ee058f4)

The only last thing you need to fix is the some Switch operator which understand from several `-f` args to bank it relay. Thats why recomend using `.startswith` method of string 

![image](https://github.com/user-attachments/assets/87c2cff6-b8c7-4a35-967d-f8634d0e2319)